pkg/renovate/job.go

```diff
@@ -146,6 +237,20 @@ func (j *JobCoordinator) Execute(ctx context.Context, tasks []*Task) error {
 								},
 							},
 						},
+						{
+							Name: "registry-secrets",
```

Comment:
When there is no such secret created (e.g. we don't have any pull secret prepared in cluster and mintmaker namespace), this will raise an error.

Here is an example how we mount a configmap only when it exists: https://github.com/konflux-ci/mintmaker/blob/8dde1c20a51bca9b74dabb42d7defc703b4432bd/pkg/renovate/job.go#L196-L225

Reply:
Fixed, thank you